lib/uniffi_backend_keys/tests/swift/test.swift

```diff
@@ -38,3 +38,10 @@ assert(unifiedSpendingKey.toUnifiedFullViewingKey()
 
 // Test ZcashAccountPrivKey.fromSeed
 assert(accountPrivKey.toBytes() == accountPrivKeyBytes)
+
+// Test ZcashExtendedPrivKey.withSeed, ending up in a private key.
+let accountExtendedPrivKey = try ZcashExtendedPrivKey.withSeed(data: seed);
+let privkey = ZcashAccountPrivKey.fromExtendedPrivkey(key: accountExtendedPrivKey);
+// This "golden" array of bytes was obtained from the original API.
+let expected_bytes: [UInt8] = [149, 166, 0, 189, 51, 113, 120, 140, 117, 138, 197, 190, 79, 199, 36, 207, 244, 33, 243, 51, 208, 246, 123, 28, 7, 59, 51, 23, 198, 47, 254, 133, 152, 104, 37, 20, 245, 17, 68, 127, 3, 242, 44, 136, 77, 60, 173, 143, 250, 10, 6, 187, 211, 123, 199, 44, 60, 30, 76, 229, 192, 202, 104, 8]
+assert(privkey.toBytes() == expected_bytes);
```

Comment:
Something to investigate: Similar to Kotlin tests, if this continues growing, maybe we should look for a more structured way of testing, in which we dont need to share the global scope for all tests.

Reply:
I'm researching if it would be possible to generate foreign tests based on Rust code - of course it wouldn't be full Rust to Kotlin/Swift/Python/Ruby transpiler, but if tests were simple enough (declare some vector, call some methods, assert equality etc.) it should be feasible to do...